engine/class_modules/sc_rogue.cpp

```diff
@@ -8225,6 +8332,75 @@ void actions::rogue_action_t<Base>::trigger_restless_blades( const action_state_
   }
 }
 
+template <typename Base>
+void actions::rogue_action_t<Base>::trigger_hand_of_fate( const action_state_t* state, bool biased, bool delivered )
+{
+  if ( !p()->talent.fatebound.hand_of_fate->ok() ) {
+    return;
+  }
+
+  if ( cast_state( state )->get_combo_points() < p()->talent.fatebound.hand_of_fate->effectN( 1 ).base_value() ) {
+    return;
+  }
+
+  auto tails_action = delivered ? p()->active.fatebound.fatebound_coin_tails_delivered : p()->active.fatebound.fatebound_coin_tails;
+
+  // No stacks of either buff or equal stacks of both buffs (thanks to only using edge case) - nothing to bias,
+  // just flip the coin
+  if ( p()->buffs.fatebound_coin_tails->total_stack() == p()->buffs.fatebound_coin_heads->total_stack() ) {
+    if ( p()->rng().roll( 0.5 ) ) {
+      // Heads
+      p()->buffs.fatebound_coin_tails->expire();
+      p()->buffs.fatebound_coin_heads->increment();
+    }
+    else {
+      // Tails
+      tails_action->execute_on_target( state->target );
+
+      p()->buffs.fatebound_coin_heads->expire();
+      p()->buffs.fatebound_coin_tails->increment();
+    }
+    return;
+  }
+
+  double odds = 0.5;
+  if (biased) {
+    // TODO: Validate how these stack with the presumed base 50/50 chance and one another
```

Comment:
I have no idea if these are additive (with either one another or the base 50/50 odds) or multiplicative. And the only way to know for sure would seem to be a *lot* of log analysis. I think the pessimistic option would be swapping these to be additive with one another but multiplicative with the base 0.5.  So
```suggestion
    // TODO: Validate how these stack with the presumed base 50/50 chance and one another
    double bias = 1.0;
      if (p()->talent.fatebound.mean_streak->ok()) {
      bias += p()->talent.fatebound.mean_streak->effectN( 1 ).percent();
    }
    if (p()->talent.fatebound.destiny_defined->ok()) {
      bias += p()->talent.fatebound.destiny_defined->effectN( 3 ).percent();
    }
    odds *= bias;
```
This works out to be 69% in favor of a match, while the current stacking is 88% - much higher.